gremlin-scala/src/test/scala/gremlin/scala/TraversalSpec.scala

```diff
@@ -356,22 +356,20 @@ class TraversalSpec extends WordSpec with Matchers {
   }
 
   "union" should {
-
     "work for traversals with the same end type" in new Fixture {
-      val traversal = graph.V(4).union(
-        __[Vertex].in.value(Age),
-        __[Vertex].in.out.value(Age)
-      )
+      val traversal: GremlinScala[Int, _] =
+        graph.V(4).union(
+          _.in.value(Age),
+          _.in.out.value(Age)
+        )
 
```

Comment:
Re lambdas: yup, that's one of the main points of gremlin-scala: helping the developer with more type safety. 

I agree that a type alias would make it easier to read, but not nessecarily that it makes it easier to understand. But I get your point and will have a think about it. Thanks!